CHANGES.md

```diff
@@ -1,6 +1,9 @@
 ## Changes in 0.9.1 (in development)
 * Fixed issue that datasets with spatial bounds could not be opened.
 * Show version as `__version__`
+* The CCI data attribute `data_type` has been renamed to `type_of_data` to
```

Comment:
What is a "CCI data attribute" and where is it used? What is the impact for users?

Reply:
I use this term to refer to attributes of CCI data. In the CCI context, there are certain attributes that many or all datasets have. In particular, a cci dataset id is built from several of these attributes. The term 'dataType' is also used as an opensearch parameter.